user_docs/en/userGuide.t2t

```diff
@@ -888,9 +888,15 @@ If input is not supported on a display which does have a braille keyboard, this
 ==== Expand to computer braille for the word at the cursor ====
 This option allows the word that is under the cursor to be displayed in non-contracted computer braille.
 
-==== Cursor Blink Rate ====
+==== Show Cursor ====
+This option allows the braille cursor to be turned on and off. It applies to all cursors including the selection indicator.
+
+==== Cursor Blink Rate (ms) ====
 This option is a numerical field that allows you to change the blink rate of the cursor in milliseconds.
 
+==== Cursor Shape ====
+This option allows you to choose the shape of the braille cursor. Shapes are defined as dot patterns. The selection indicator is not affected by this option.

```

Comment:
nit: I think "Shapes are defined as dot patterns" is a bit confusing for some, even though I know exactly what you mean. I've just put dot pattern in parentheses like so:

> This option allows you to choose the shape (dot pattern) of the braille cursor.